.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+    - 3.4
+    - 3.5
+    - 3.6
+script:
+    - python -m unittest
```

Comment:
Don't you want this to be `python run_tests.py`?

Reply:
FIxed